torch/csrc/jit/mobile/nnc/aot_compiler.cpp

```diff
@@ -0,0 +1,120 @@
+// (c) Facebook, Inc. and its affiliates. Confidential and proprietary.
+
+#include "aot_compiler.h"
+
+#include <ATen/Functions.h>
+#include <ATen/NativeFunctions.h>
+#include <torch/csrc/jit/ir/ir.h>
+#include <torch/csrc/jit/jit_opt_limit.h>
+#include <torch/csrc/jit/passes/constant_propagation.h>
+#include <torch/csrc/jit/passes/dead_code_elimination.h>
+#include <torch/csrc/jit/tensorexpr/ir.h>
+#include <torch/csrc/jit/tensorexpr/ir_simplifier.h>
+#include <torch/csrc/jit/tensorexpr/kernel.h>
+#include <torch/csrc/jit/tensorexpr/llvm_codegen.h>
+#include <torch/csrc/jit/tensorexpr/loopnest.h>
```

Comment:
Some includes here are unnecessary:
```
ir_simplifier.h
loopnest.h
llvm_codegen.h
jit_opt_limit.h
```
It's a good practice only include what's used in the file. Also, please run clang-format to sort includes alphabetically.